### DISCUSSION TITLE
Bump Pydantic to 1.9.0

### DISCUSSION THREAD

**Discussion by yudjinn** [collerek/ormar#509] [Q&A] [answered]
Hey, I have other libraries that depend on 1.9.0 of pydantic; I dont see it breaking anything in ormar; is it possible to update?

**Comment by collerek** [comment, marked as answer]
There is one breaking internal import, already fixed in #508 
Waiting for the release of databases that fixes sqlite parameters: https://github.com/encode/databases/pull/441

**Reply by yudjinn** [reply]
awesome, thanks!
